test/e2e/Makefile

```diff
@@ -47,8 +48,12 @@ KUSTOMIZE := $(TOOLS_BIN_DIR)/kustomize
 OUTPUT_DIR := _output/$(GOOS)/$(GOARCH)/bin
 GINKGO_FOCUS ?=
 VELERO_CLI ?=$$(pwd)/../../_output/bin/$(GOOS)/$(GOARCH)/velero
-VELERO_IMAGE ?= velero/velero:main
+VELERO_IMAGE ?= $(DEFAULT_VELERO_IMAGE)
+VELERO_VERSION ?= $(word 2, $(default_velero_image_temp))
```

Comment:
Use `VERSION` instead

Reply:
It's the same issue with the upper one .
-------
It's not needed since I've changed code to pass version from main Makefile to E2E Makefile, so don't have to parse from this default value.